test/fixtures/output/schema-registry/help.golden

```diff
@@ -7,14 +7,13 @@ Aliases:
   schema-registry, sr
 
 Available Commands:
-  cluster       Manage Schema Registry cluster.
-  compatibility Validate schema compatibility.
-  config        Manage Schema Registry configuration.
-  dek           Manage Schema Registry Data Encryption Keys (DEKs).
-  exporter      Manage Schema Registry exporters.
-  kek           Manage Schema Registry Key Encryption Keys (KEKs).
-  schema        Manage Schema Registry schemas.
-  subject       Manage Schema Registry subjects.
+  cluster     Manage Schema Registry cluster.
+  config      Manage Schema Registry configuration.
```

Comment:
We have a ticket to rename this to "configuration", right?

Reply:
Doesn't look like it. I'll make a ticket for it and bundle this change in with this PR.